commands/init.ts

```diff
@@ -168,24 +185,30 @@ exports.handler = async options => {
 };
 
 exports.builder = yargs => {
-  yargs.options({
-    auth: {
-      describe: i18n(`${i18nKey}.options.auth.describe`),
-      type: 'string',
-      choices: [
-        `${PERSONAL_ACCESS_KEY_AUTH_METHOD.value}`,
-        `${OAUTH_AUTH_METHOD.value}`,
-      ],
-      default: PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
-      defaultDescription: i18n(`${i18nKey}.options.auth.defaultDescription`, {
-        defaultType: PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
-      }),
-    },
-    account: {
-      describe: i18n(`${i18nKey}.options.account.describe`),
-      type: 'string',
-    },
-  });
+  yargs
+    .options({
+      auth: {
+        describe: i18n(`${i18nKey}.options.auth.describe`),
+        type: 'string',
+        choices: [
+          `${PERSONAL_ACCESS_KEY_AUTH_METHOD.value}`,
+          `${OAUTH_AUTH_METHOD.value}`,
+        ],
+        default: PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
+        defaultDescription: i18n(`${i18nKey}.options.auth.defaultDescription`, {
+          defaultType: PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
+        }),
+      },
+      account: {
+        describe: i18n(`${i18nKey}.options.account.describe`),
+        type: 'string',
+      },
+      useHiddenConfig: {
```

Comment:
When we did the command audit, did we land on using `kebab-case` for the options?

Reply:
We did! Will change that up.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1123/commits/ce0d196390dbb88f81f8b820385c6d0dfb12e783